apps/uci/web/src/components/common/fullscreen-loader/index.tsx

```diff
@@ -1,18 +1,18 @@
 import React, { FC } from 'react';
 import styles from './index.module.css';
 import { Modal, ModalContent, ModalOverlay } from '@chakra-ui/react';
+import { config } from '@/config';
 
-const FullScreenLoader: FC<{ loading: boolean }> = ({ loading }) => (
+export const FullScreenLoader: FC<{ loading: boolean }> = ({ loading }) => (
 
 	<Modal isCentered isOpen={loading} onClose={() => null} >
 		<ModalOverlay
-			bg='blackAlpha.300'
-			backdropFilter='blur(10px) hue-rotate(90deg)'
+			bg={config.fullscreeenLoader.modalBgColor}
```

Comment:
little confusing here, everything is imported from config ,for theme or style it should be style